src/runtime/components/forms/Textarea.vue

```diff
@@ -154,6 +154,7 @@ export default defineComponent({
         }
 
         textarea.value.rows = props.rows
+        textarea.value.style.overflow = 'hidden'
```

Comment:
Just to be safe we might want to preserve the original value and set it back once we're done:
```suggestion
        const overflow = textarea.value.style.overflow
        textarea.value.style.overflow = 'hidden'
```